Gemfile

```diff
@@ -0,0 +1,8 @@
+source 'https://rubygems.org'
+
+gem 'rake'
+gem 'neat'
+gem 'kramdown'
+gem 'jekyll'
+gem 'jekyll-paginate'
+gem 'jekyll-redirect-from'
```

Comment:
```
ruby --version
ruby 2.0.0p353 (2013-11-22 revision 43784) [x86_64-linux]
```

Presumably you will now be able to compromise my machine via my hilariously old version of ruby :stuck_out_tongue: